<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>re_3_store</class>
 <widget class="QWidget" name="re_3_store">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>商品编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="numlineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="quantitylineEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="indexbytime_pbtn">
        <property name="text">
         <string>商品查询（临期）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QPushButton" name="indexbyquantity_pbtn">
        <property name="text">
         <string>商品查询（数量）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QPushButton" name="destroy_pbtn">
        <property name="text">
         <string>销毁</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
